bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/update/update.xml

```diff
@@ -9,5 +9,18 @@
 			</add-channel>
 		</instruction-set>
 	</thing-type>
-
+	<thing-type uid="openwebnet:bus_energy_meter">
+		<instruction-set targetVersion="1">
+			<add-channel id="energyToday">
+				<type>openwebnet:energyToday</type>
+			</add-channel>
+		</instruction-set>
+	</thing-type>
+	<thing-type uid="openwebnet:bus_energy_meter">
+		<instruction-set targetVersion="1">
+			<add-channel id="energyThisMonth">
+				<type>openwebnet:energyThisMonth</type>
+			</add-channel>
+		</instruction-set>
+	</thing-type>
```

Comment:
You also need to add the property `thingTypeVersion` on thing type bus_energy_meter.

And all your code can be reduced to:
```
	<thing-type uid="openwebnet:bus_energy_meter">
		<instruction-set targetVersion="1">
			<add-channel id="energyToday">
				<type>openwebnet:energyToday</type>
			</add-channel>
			<add-channel id="energyThisMonth">
				<type>openwebnet:energyThisMonth</type>
			</add-channel>
		</instruction-set>
	</thing-type>
```

Reply:
aaah.. good! I will apply this

Reply:
thingTypeVersion property: which value?  maybe 1?